apps/prairielearn/src/migrations/20231031163354_client_fingerprints__create.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE
+  client_fingerprints (
+    id BIGSERIAL PRIMARY KEY,
+    user_id BIGINT REFERENCES users (user_id),
+    session_id TEXT REFERENCES user_sessions (session_id),
+    ip_address INET NOT NULL,
+    user_agent TEXT NOT NULL,
+    accept TEXT NOT NULL,
+    accept_language TEXT NOT NULL
```

Comment:
Rather than making these `TEXT` fields, we should consider truncating these to a known length (`VARCHAR(255)`?).

Reply:
I have updated these fields to `VARCHAR(255)` in commit [b135adf](https://github.com/PrairieLearn/PrairieLearn/pull/8798/commits/b135adfd92d740104813013c22b9b36e61ec23bd).